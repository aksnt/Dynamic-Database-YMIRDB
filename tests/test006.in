SET a 2 1 3 2 1 1 1 2
SORT A
GET A
UNIQ A
GET A
SET B 1 2 A
SORT B
UNIQ B
REV B