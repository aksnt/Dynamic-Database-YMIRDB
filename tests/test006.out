> ok

> ok

> [1 1 1 1 2 2 2 3]

> ok

> [1 2 3]

> ok

> not permitted on general entries

> not permitted on general entries

> not permitted on general entries
